core/src/main/java/hudson/FilePath.java

```diff
@@ -482,21 +483,39 @@ public int zip(OutputStream out, DirScanner scanner) throws IOException, Interru
      *      is archived.
      */
     public int archive(final ArchiverFactory factory, OutputStream os, final DirScanner scanner) throws IOException, InterruptedException {
+        Consumer<String> c = (Consumer<String> & Serializable)(s) -> {};
+        return archive(factory, os, scanner, c);
+    }
+
+    /**
+     * Archives this directory into the specified archive format, to the given {@link OutputStream}, by using
```

Comment:
Could you complete the javadoc with the @params, @since an @throw sections please? It will help clients of this new method

Reply:
some of them are describe above, and I just copied and updated the javadoc from the overloaded method.